patches/minecraft/net/minecraft/client/gui/GuiGameOver.java.patch

```diff
@@ -0,0 +1,15 @@
+--- a/net/minecraft/client/gui/GuiGameOver.java
++++ b/net/minecraft/client/gui/GuiGameOver.java
+@@ -40,6 +40,12 @@
+       GuiButton guibutton = this.func_189646_b(new GuiButton(1, this.field_146294_l / 2 - 100, this.field_146295_m / 4 + 96, s1) {
+          public void func_194829_a(double p_194829_1_, double p_194829_3_) {
+             if (GuiGameOver.this.field_146297_k.field_71441_e.func_72912_H().func_76093_s()) {
++               GuiGameOver.this.field_146297_k.field_71441_e.func_72882_A(); //forge begin - fixes MC-30646 and MC-96521
```

Comment:
To reduce the patch size and to make the code more readable, please put this code into a new method in `net.minecraftforge.common.ForgeHooks` and then call it from here.

Reply:
done.